Prevention Act;Prevents sandmining, scamming and trapping.html

```diff
@@ -0,0 +1,18 @@
+<h2>Prevention</h2>
+<h3>Sandmining</h3>
+<p>
+    Do not sand mine, if a player does sand mine the military may be deployed
+    and if it’s severe the criminal may be given a fine of 1g per chunk
+    affected.
+</p>
+<h3>Scams<h3>
+<p>
+    Do not scam people, do trades fairly. If unsure of how much
+    something sells for, ask someone who is trustworthy and would know.
+<p>
+<h3>Trapping<h3>
+<p>
+    If a player trap a citizen of Sudan they will be required to give them their stuff
+    back and if they have a high ranking position in a town/nation it may be 
+    removed.
```

Comment:
Should be more specific, what is a high ranking town/nation position?
We probably don't want to forcibly remove a town position, but we could advise mayors to do so.